java-kotlin-aot/README.md

```diff
@@ -13,9 +13,9 @@ This repository contains the code for a demo application for [GraalVM](graalvm.o
 This is a simple Java/Kotlin application, where a Java method accesses a String from Kotlin and calls a Kotlin function, which later accesses a String from a Java class.
```

Comment:
Please do the following edits in README.md:
1. The host command `-H:+ReportUnsupportedElementsAtRuntime` should be replaced with this stable command ` --report-unsupported-elements-at-runtime`. 
 2. Change the title demo to "Java and Kotlin Interoperability Demo"

Reply:
3. The option to specify the image name is `-o helloworld`. Replace in the command instead of `-H:Name=helloworld`.